tests/beaker_tests/cisco_tacacs_server/tacacsserver_provider_nondefaults.rb

```diff
@@ -100,7 +100,7 @@
                              { 'ensure'              => 'present',
                                'timeout'             => '50',
                                'deadtime'            => '0',
-                               'encryption_password' => add_quotes('WXYZ12'),
```

Comment:
Go ahead and remove the `add_quotes` method assuming it's not used anywhere else.